cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -158,6 +158,16 @@
             margin: 0;
         }
 
+        .o-footer_col:nth-child(n+2) {
+            .o-footer_list {
+                .respond-to-max( @bp-xs-max, {
```

Comment:
Should the media query go on the outside of this entire block?

Reply:
Yeah I think that's probably cleaner less, fixed in 534d1f2.